win32_event_log/manifest.json

```diff
@@ -38,6 +38,9 @@
     },
     "logs": {
       "source": "windows.events"
+    },
+    "dashboards": {
+      "Windows Event Log Overview": "assets/dashboards/windows_event_log_overview.json"
```

Comment:
the `short_name` you chose (`"Windows Event Log Overview"`) is invalid - should be `snake_case`, e.g. `windows_event_log_overview`

Reply:
based on this [doc](https://datadoghq.atlassian.net/wiki/spaces/DB/pages/3655009149/Internal+Integration+Dashboard+PR+Review+guide), I think `short_name` should match `assets/dashboard/{short_name}.json`